### PR TITLE
add a new target CADDXF4

### PR DESCRIPTION
**Pull request by hakrc1**
Hi
1. Client changed board name.
2. CADDX TECHNOLOGY (SHENZHEN) CO., LTD    andy@caddxfpv.com

### CODE REVIEW

File: configs/NETURON_F405/config.h [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+#define MANUFACTURER_ID   CADX
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_ACCGYRO_BMI270
+#define USE_BARO_BMP280
```
Comment (haslinghuis) [inline configs/NETURON_F405/config.h:36]:
```suggestion
#define USE_BARO
#define USE_BARO_BMP280
```

File: configs/NETURON_F405/config.h [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    TIMER_PIN_MAP( 6, PB0 , 2,  0) \
+    TIMER_PIN_MAP( 7, PB1 , 2,  0) \
+    TIMER_PIN_MAP( 8, PB3 , 1,  0) \
+    TIMER_PIN_MAP( 9, PB8 , 1,  0) \
+    TIMER_PIN_MAP(10, PB9 , 2, -1)
+
+
+#define ADC3_DMA_OPT        0
+
+#define USE_BARO
```
Comment (haslinghuis) [inline configs/NETURON_F405/config.h:104]:
```suggestion
```

**Comment by nerdCopter** [comment]
`NETURON` not `NEUTRON` ??

**Comment by hakrc1** [comment]
The client has determined that using CADDXF4 board name

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**